internal/datastore/mysql/revisions.go

```diff
@@ -153,7 +156,10 @@ func (mds *Datastore) checkValidTransaction(ctx context.Context, revisionTx uint
 		return false, false, fmt.Errorf(errCheckRevision, err)
 	}
 
-	span.AddEvent("DB returned validTransaction checks")
+	span.AddEvent(otelconv.EventDatastoreMySQLTransactionValidated, trace.WithAttributes(
```

Comment:
Can we go one step further and classify events as "internal" vs "external", and name them accordingly, like `spicedb-internal.` vs `spicedb.`? This will allow us to filter traces in a nice way down the road.